src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -3750,7 +3750,7 @@ public static int M(out int a)
                 );
         }
 
-        [Fact]
+        [Fact(Skip = "PROTOTYPE(semi-auto-props): WRN_UnreadRecordParameter is generated for no reason. Investigate why this happened.")]
```

Comment:
>Skip = "PROTOTYPE(semi-auto-props): WRN_UnreadRecordParameter is generated for no reason. Investigate why this happened." [](http://example.com/codeflow?start=14&length=121)

Please do not skip legacy tests. Update the baseline and add a PROTOTYPE comment explaining what we don't like or question about the change.

Reply:
Un-skipped.